tests/PHPStan/Analyser/data/integer-range-types.php

```diff
@@ -228,10 +228,10 @@ public function math($i, $j, $z, $pi, $r1, $r2, $rMin, $rMax, $x, $y) {
 		assertType('int<min, 15>', $j * $rMin);
 		assertType('int<5, max>', $j * $rMax);
 
-		assertType('int<-19, 13>', $r1 + $z);
-		assertType('int<-2, 30>', $r1 - $z);
-		assertType('int<-20, 30>', $r1 * $z);
-		assertType('float', $r1 / $z);
+		assertType('int<-19, -10>|int<2, 13>', $r1 + $z);
+		assertType('int<-2, 9>|int<21, 30>', $r1 - $z);
+		assertType('int<-200, -20>|int<1, 30>', $r1 * $z);
+		assertType('float|int<0, 10>', $r1 / $z);
```

Comment:
after I added coverage for the test-cases in https://github.com/phpstan/phpstan-src/pull/652 I realized that these operations here can be expressed more precily. and therefore we have the changes here